scopes/categories.md

```diff
@@ -0,0 +1,32 @@
+
+ - Core application scope types
+ - Standard application scope types
+ - Extended application scope types
+
+## Core application scope types
+
+Core application scope types define grouping constructs for basic runtime behavior. They have the following characteristics:
+
+ - Core application scope types MUST be in the `core.oam.dev` namespace.
+ - Core application scope types MUST all be supported by any implementation of the specification. They may be implemented by any available feature in the platform, but they MUST be implemented according to the specification of each core application scope type.
+ - Instances of core *workload* types MUST be deployed into an instance of each core application scope type. 
+ - Runtimes MUST provide a default "root" application scope instance for each core application scope type.
+ - Runtimes MUST deploy each component instances into default "root" application scope instances when an application configuration does not otherwise specify an application scope for a component instance.
+
+ The following core application scope types are defined by this specification:
+
+| Name | Type | Description | 
+|-----------|------|----------|
+|Network|`core.oam.dev/v1alpha2.Network` | This scope groups components into a network subnet boundaries and defines the general runtime networking model. Network definitions, rules, and policies are described by the infrastructure's network or SDN.
+|Health|`core.oam.dev/v1alpha2.Health` | This scope groups components into an aggregate health group. The aggregate health of the constituent components within the group supplies information to upgrade and rollback mechanisms.  |
+
+## Standard application scope types
+
+Standard application scope types MUST be defined in `standard.oam.dev` group. These application scopes provide operational functionality that is commonly used by application operators and implemented by most runtimes. Runtimes are RECOMMENDED to use the standard application scope definitions as defined in this specification when providing equivalent operational functionality to those listed in the standard application scope definitions. In other words, if a runtime is implementing application scope with behavior `foo` and a standard application scope definition for behavior `foo` exists, the runtime SHOULD use the standard `foo` application scope definition. Application operators that intend to maximize portability of their applications should use these application scope definitions when available. Although this does not _guarantee_ portability of an application, it is designed to increase portability across runtimes.
+
+## Extended application scope types
+
+Extended application scopes are optional per runtime, meaning that each runtime may choose which extended scopes are supported. It is expected that runtimes will implement extended scopes as they need. In this version of the spec, allowing user-defined extended application scope types is not supported.
```

Comment:
The workload PR keeps those in the main spec while just move the definitions of them out. I wonder if that makes sense?